handlers/accesscontrol/access-control.go

```diff
@@ -61,16 +62,36 @@ type HitRecords struct {
 	mux  sync.Mutex
 }
 
+type RefreshIntervalRecord struct {
+	RefreshInterval int32
+	LastRefresh     time.Time
+}
+
+type RefreshInterval struct {
+	data map[string]*RefreshIntervalRecord
+}
+
+type GateConfig struct {
+	MaxAge               int32 `json:"max_age"`
+	StaleWhileRevalidate int32 `json:"stale_while_revalidate"`
```

Comment:
could you change these to `time.Duration`?

Reply:
Don't want to touch old acl code so close to trovo launch, can do it a followup pr, if that's fine to you